cockatrice/src/carddatabasemodel.h

```diff
@@ -12,7 +12,7 @@ class FilterTree;
 class CardDatabaseModel : public QAbstractListModel {
     Q_OBJECT
 public:
-    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, CMCColumn };
+    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, ColorColumn };
```

Comment:
Fair enough. Leave it off then.

On Thu, Jun 23, 2016, 1:59 PM Zach <notifications@github.com> wrote:

> In cockatrice/src/carddatabasemodel.h
> <https://github.com/Cockatrice/Cockatrice/pull/2021#discussion_r68283757>:
>
> > @@ -12,7 +12,7 @@ class FilterTree;
> >  class CardDatabaseModel : public QAbstractListModel {
> >      Q_OBJECT
> >  public:
> > -    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, CMCColumn };
> > +    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, ColorColumn };
>
> I removed CMC because it wasn't used anywhere else in the codebase, it was
> randomly added to the column enum at some point, but not referenced. Also,
> I really like @tooomm <https://github.com/tooomm>'s idea a lot. Open a
> ticket on that and ping ctrlaltca
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/2021/files/79fc24769a366b9779c2be4ca2b969a19600636f#r68283757>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAA5NK9p3_S4Zg5edZAohEykc9AiimX5ks5qOsmPgaJpZM4I6W50>
> .
>